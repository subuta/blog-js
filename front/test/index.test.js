/* global describe, it */
import React from 'react';
import { create } from 'react-test-renderer';

const assert = require('assert');

describe('example', () => {
  it('should be rendered correctly', () => {
    const tree = create(<span>fuga</span>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
