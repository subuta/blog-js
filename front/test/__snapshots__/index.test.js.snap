// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example should be rendered correctly 1`] = `
<span>
  fuga
</span>
`;
